### PR TITLE
D17 View Orderbook

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: gui/README.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
 ## Development
 
 ```sh
-REACT_APP_API_HOST="http://localhost:3001/api" REACT_APP_LCD_API_HOST="http://localhost:1317" yarn start
+REACT_APP_API_HOST="http://localhost:3001/api" REACT_APP_LCD_API_HOST="http://localhost:3001/api/lcd" yarn start
```
Comment (musnit) [inline gui/README.md]: could add npm run scripts for convenience

File: gui/src/organisms/Orderbook/View.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  }
+`
+
+const Icon = styled.img`
+  float: right;
+  margin-top: 5px;
+`
+
+// TODO replace with dynamic function by Haneef
+const getInUsd = (coin: ICoin) => {
```
Comment (haneefghanim) [inline gui/src/organisms/Orderbook/View.tsx]: 👍

**Comment by philipstanislaus** [comment]
Awesome, thanks for the review @musnit @haneefghanim state mgmt is probably necessary in this sprint already – we have global state of which tokens are selected, for example. 

Let's see who first touches functionality that depends on chosen token pairs.

**Comment by philipstanislaus** [comment]
Just pulled the env variables into package.json scripts

**Pull request merged**